source/publishing_data/05_processing_data/processors/expression.rst

```diff
@@ -1,273 +1,352 @@
-   * * Two operands
-     * add_days, add_hours, add_minutes, add_months, add_seconds, add_years, contains, day, dayofweek, distance, div, endswith, gcd, geopoint2d_towgs84, geoshape_towgs84, hour, match, max, microsecond, min, minute, mod, month, pow, quarter, quartertodaterange, random, round, second, startswith, substring, week, year
-   * * Three operands
-     * center, datediff, ljust, replace, rjust, substring
+   * * Operator
+     * Description
+     * Example
+   * * ``-``
+     * Prefix that negates the following value
+     * ``- 4``, ``- [expression]``
+   * * ``not``, ``!`` (not)
+     * Boolean operator that inverts the following condition
+     * * ``not true`` or  ``!true`` returns ``false``
+       * ``not 4 > 5`` returns ``true``
+       * ``!(5 <= 10)`` returns ``false``
+   * * ``!`` (factorial)
+     * Suffix that computes the factorial of an expression
+     * ``3!`` returns ``1*2*3``
+
+Binary operators
+^^^^^^^^^^^^^^^^
 
+.. list-table::
+   :header-rows: 1
 
-Examples
---------
+   * * Operator
+     * Description
+     * Example
+   * * ``+``, ``-``, ``*``, ``/``, ``%``, ``//``, ``^``
+     * Arithmetic operators: add, subtract, multiply, divide, modulo, euclidian division, power
+     * * ``2 + 3`` returns ``5``
+       * ``5 - 8`` returns ``-3``
+       * ``9 * 3`` returns ``27``
+       * ``15 / 6`` returns ``2.5``
+       * ``15 % 6`` returns ``3``
+       * ``15 // 6`` returns ``2``
+       * ``2 ^ 3`` returns ``8``
+       * ``column_1 + column2``
+   * * ``and``, ``&&``, ``or``, ``||`` (return a boolean)
+     * Boolean operators: and, or
+     * * ``true and true`` returns ``true``
+       * ``true && false`` returns ``false``
+       * ``true or false`` returns ``true``
+       * ``false || false`` returns ``false``
+   * * ``>``, ``<``, ``>=``, ``<=``, ``==``, ``!=``
+     * Comparison operators: greater than, lower than, greater or equal to, lower or equal to, equal to, different than return a boolean
+     * * ``3 < 4`` returns ``true``
+       * ``5 >= 10`` returns ``false``
+   * * ``&``
+     * String concatenation operator
+     * ``'Hello' & ' world'`` returns ``'Hello world'``
+
+Ternary operator
+^^^^^^^^^^^^^^^^
+
+The expression ``[condition] ? [result if true] : [result if false]`` is called the ternary operator, and allows to return different results depending on a condition.
 
 .. admonition:: Note
    :class: note
 
-   Keep in mind that for each expression, the examples are presented with actual numbers, but they can be replaced with the processed dataset's fields technical identifiers.
+   A ternary expression can be read as "if [condition] then [result if true] else [result if false]".
+
+Examples:
+
+- ``true ? 'hello' : 'goodbye'`` returns ``'hello'``
```

Comment:
the examples with ` ``true`` ` need quotes around true/false to work; ` ``'true'`` `

Reply:
👍